pangeo_forge_recipes/writers.py

```diff
@@ -99,19 +99,29 @@ def write_combined_reference(
     full_target: FSSpecTarget,
     concat_dims: List[str],
     output_file_name: str,
-    target_options: Optional[Dict] = {"anon": True},
-    remote_options: Optional[Dict] = {"anon": True},
-    remote_protocol: Optional[str] = None,
     refs_per_component: int = 1000,
     mzz_kwargs: Optional[Dict] = None,
 ) -> zarr.storage.FSStore:
     """Write a kerchunk combined references object to file."""
     file_ext = os.path.splitext(output_file_name)[-1]
     outpath = full_target._full_path(output_file_name)
 
+    import ujson  # type: ignore
+
+    # full_target in tests are strings so accommodate
+    # setup storage options for fsspec.MultiZarrToZarr and fsspec.ReferenceFileSystem
+    if isinstance(full_target, FSSpecTarget):
```

Comment:
I might be missing something here. Added to accommodate tests

I don't like accommodating test paths inside the code like this. Is there a reason we use `conftest.py:tmp_target_url` instead of `conftest.py:tmp_target` that actually returns an `FSSPecTarget`?